Formula/avfs.rb

```diff
@@ -4,6 +4,11 @@ class Avfs < Formula
   url "https://downloads.sourceforge.net/project/avf/avfs/1.1.4/avfs-1.1.4.tar.bz2"
   sha256 "3a7981af8557f864ae10d4b204c29969588fdb526e117456e8efd54bf8faa12b"
 
```

Comment:
```suggestion
  license "GPL-2.0-only"

```

 The `COPYING` file is GPLv2 and the `README` states: "AVFS is (C) under the GNU GPL (see the file COPYING). The shared library is (C) under the GNU LGPL (see the file COPYING.LIB)."

The files in the `src` directory contain a leading comment like:

```
AVFS: A Virtual File System Library
Copyright (C) 1998  Miklos Szeredi <miklos@szeredi.hu>

This program can be distributed under the terms of the GNU GPL.
See the file COPYING.
```

There are a few files that use the "or...later" language but the files in the `src` directory don't, so `GPL-2.0-only` may be appropriate here insomuch as it's not explicitly `-or-later`. Granted, I'm not a license expert.

Reply:
I was thinking to address the license in a separate PR. But I am ok to cover in this one as well.

Reply:
yeah, license is a bit complicated, and I create [a separate PR](https://github.com/Homebrew/homebrew-core/pull/116649) for it